src/components/ChatComponent/MessageBubble.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+const StyledMessageBubble = styled.div`
+  width: 400px;
+  height: 110px;
+  padding: 15px 20px;
+  display: flex;
+  align-items: center;
+  margin: 10px;
```

Comment:
In general there're some inconsistencies with colour and size values. Sometimes rem, sometimes px, pick one as a group and use that :)